src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomLocationMetadataMapper.xml

```diff
@@ -179,29 +179,32 @@
 	<select id="selectLocationHierachy"
 			resultMap="DetailsResultMap">
 		select lm.geojson_id identifier ,lm."name" ,lm.parent_id, 'false' voided, lm.location_id,string_agg(lt."name",',') tags,
```

Comment:
Introduce a flag to mark location tags optional, if they are not required grouping can be avoided.

Select string_agg(lt."name",',') tags if tags are required